source/reference/options.txt

```diff
@@ -76,6 +76,36 @@ General Options
 
    Returns the |mdb-shell| release number.
 
+.. _mongosh-stable-api-options:
+
+Stable API Options
+------------------
+
+.. option:: --apiVersion <version number>
+
+   Specifies the :ref:`apiVersion <api-version-desc>`. ``"1"`` is 
+   currently the only supported value.
+
+.. option:: --apiStrict
+
+   Specifies that the server will respond with :ref:`APIStrictError 
+   <api-strict-resp>` if your application uses a command or behavior 
+   outside of the :ref:`Stable API <stable-api>`.
```

Comment:
Should we mention that when this is not specified then mongosh defaults to a non-strict client? I think it's probably fine as is as well.